src/inmanta/data/__init__.py

```diff
@@ -2927,6 +2927,24 @@ async def get_by_id(
             return result[0]
         return None
 
+    async def release_version_lock(self, *, shared: bool = False, connection: asyncpg.Connection) -> None:
```

Comment:
Hah, I hadn't even noticed the ambiguity of this name: I think `release_version_lock` is to be interpreted as "acquire a lock for the `release_version` context", rather than "release a lock". But the ambiguity could be a concern.

Reply:
mhh, it is the lock on release version, how do I formulate that?

Reply:
What about `acquire_release_version_lock`?

Reply:
changed